pkg/reconciler/revision/resources/queue.go

```diff
@@ -379,6 +379,9 @@ func makeQueueContainer(rev *v1.Revision, cfg *config.Config) (*corev1.Container
 		}, {
 			Name:  "ENABLE_HTTP2_AUTO_DETECTION",
 			Value: strconv.FormatBool(cfg.Features.AutoDetectHTTP2 == apicfg.Enabled),
+		}, {
+			Name:  "ROOT_CA",
```

Comment:
It looks like this is normally `SSL_CERT_DIR` / `SSL_CERT_FILE` for go / unix TLS system root pool.

Is this intended to establish a separate CA pool, or should this be adding to an existing pool?

Reply:
Adding to the CA Pool (if exist) and creating new one with only this Cert if no pool exists.

Reply:
I was trying to figure out if this was going to use / influence https://pkg.go.dev/crypto/x509#SystemCertPool

It sounds like this will be a separate pool?